app/src/main/java/chat/rocket/android/contacts/ContactsFragment.kt

```diff
@@ -113,8 +127,8 @@ class ContactsFragment : Fragment() {
 
         val searchItem = menu.findItem(R.id.action_search)
         searchView = searchItem?.actionView as? SearchView
-        searchView?.setIconifiedByDefault(false)
-        searchView?.maxWidth = Integer.MAX_VALUE
+        if (Constants.WIDECHAT) {
+        setupWidechatSearchView()}
```

Comment:
can you return inside the Constants.WIDECHAT block please, after setupWidechatSearchView()?  After that there is no reason to move on in this method.  Thanks.

Reply:
I did not get what you are saying

Reply:
Does this make sense?

if (Constants.WIDECHAT) {
        setupWidechatSearchView()
        return
    }